exec/rpc/src/main/java/org/apache/drill/exec/rpc/RequestIdMap.java

```diff
@@ -84,7 +115,7 @@ public void operationComplete(ChannelFuture future) throws Exception {
       if (!future.isSuccess()) {
         removeFromMap(coordinationId);
         if (future.channel().isActive()) {
-           throw new RpcException("Future failed") ;
+          throw new RpcException("Future failed");
```

Comment:
This has existed since this commit (Sept 2013): https://github.com/apache/drill/commit/3f101aabd05f174e8bfc9dbdd2590066a1e937f4

I'm not inclined to change it in this patch as it seems out of scope. The change probably makes sense but let's explore in a different jira to ensure we're thinking through the ramifications.